packages/@webex/plugin-meetings/test/unit/spec/rtcMetrics/index.ts

```diff
@@ -13,7 +13,7 @@ describe('RtcMetrics', () => {
   beforeEach(() => {
     clock = sinon.useFakeTimers();
     webex = new MockWebex();
-    metrics = new RtcMetrics(webex, 'mock-meeting-id');
+    metrics = new RtcMetrics(webex, 'mock-meeting-id', 'mock-correlation-id');
```

Comment:
would be good to have a unit test that verifies that all the right values like correlation id, meeting id, user id are sent correctly - these checks could be added to the existing 'sendMetrics should send a webex request' test

Reply:
Updated.